datasets/modeled/industry/dataset.json5

```diff
@@ -0,0 +1,118 @@
+{
+  dataset_id: "decarb_2023_industry",
+  dataset_type: "modeled",
+  data_source: "ILSST_IGATE-E_GCAM",
+  sector_description: "industrial_manufacturing",
+  description: "Load shapes modeled for the manufacturing sector created using LBNL Industrial Load Shape Simulation Tool, national facility demand estimates from IGATE-E, and forecasts from GCAM.",
+  origin_creator: "Sarah Smith",
+  origin_organization: "LBL",
+  origin_contributors: [
+    "Samanvitha Murthy",
+    "Ikenna Okeke",
+    "Colin McMillian",
+  ],
+  origin_project: "DECARB Task 6 Year 1",
+  origin_date: "2024-02-06",
+  origin_version: "0.1.0",
+  source: "LBL SIELS (Simulator for Industrial Electricity Load Shapes)",
+  data_classification: "low",
+  tags: [
+    "industrial",
+  ],
+  trivial_dimensions: [
+    "sector",
+    "weather_year",
+  ],
+  data_schema: {
+    table_format: {
+      format_type: "pivoted",
+      pivoted_dimension_type: "metric",
+    },
+    data_schema_type: "one_table",
+  },
+  use_project_geography_time_zone: true,
+  dimensions: [
+    {
+      type: "geography",
+      name: "State",
+      display_name: "State",
+      file: "dimensions/states.csv",
+      module: "dsgrid.dimension.standard",
+      class: "State",
+      description: "US States and DC",
+    },
+    {
+      type: "metric",
+      name: "Industry End Uses",
+      display_name: "Industry End Uses",
+      file: "dimensions/enduses.csv",
+      module: "dsgrid.dimension.standard",
+      class: "EnergyEndUse",
+      description: "MECS Industry End Uses",
+    },
+    {
+      type: "model_year",
+      name: "DECARB Model Years",
+      display_name: "DECARB 2023 Industry Model Years",
+      file: "../../../project/dimensions/model_years.csv",
+      module: "dsgrid.dimension.standard",
+      class: "ModelYear",
+      description: "Even years 2024-2030, then every 5 years through 2050.",
```

Comment:
If this is exactly the same as the project model_year dimension, all the fields should match (e.g., display_name and description).